src/themes/dspace/styles/_global-styles.scss

```diff
@@ -17,7 +17,7 @@
     background-color: var(--bs-primary);
   }
 
-  h4 {
+    label {
     font-size: 1.1rem
```

Comment:
This style change might need to be reverted as I think you may need to revert some of the `<label>` tags, unless you see a different way to fix the accessibility warnings from WAVE.